profiles/sssd/system-auth

```diff
@@ -27,11 +28,13 @@ account     sufficient                                   pam_localuser.so
 account     sufficient                                   pam_usertype.so issystem
 account     [default=bad success=ok user_unknown=ignore] pam_sss.so
 account     required                                     pam_permit.so
+-account    sufficient                                   pam_systemd_home.so                                    {include if "with-homed"}
```

Comment:
The `pam_unix.so` apparently needs to be changed to `sufficient` to make this work - again I can't say why. So maybe something like the following would work:
```
account required pam_unix.so {exclude if "with-homed"}
account sufficient pam_unix.so {include if "with-homed"}
```